internal/incident/incident.go

```diff
@@ -385,18 +376,24 @@ func (i *Incident) notifyContacts(ev *event.Event, causedBy types.Int) error {
 
 			causedByHistoryId, err := i.AddEscalationTriggered(state, history)
 			if err != nil {
-				i.logger.Errorln(err)
+				i.logger.Errorw(
+					"Failed to add escalation triggered history", zap.String("rule", r.Name),
+					zap.String("escalation", escalation.DisplayName()), zap.Error(err),
+				)
 
-				return err
+				return errors.New("failed to add escalation triggered history")
 			}
```

Comment:
`AddEscalationTriggered()` actually writes to both `incident_rule_escalation_state` and `incident_history` so that log message can be misleading. Probably better move the logging inside that function with two different messages.